modules/default/updatenotification/updatenotification.js

```diff
@@ -9,11 +9,34 @@ Module.register("updatenotification", {
 		updateInterval: 10 * 60 * 1000, // every 10 minutes
 		refreshInterval: 24 * 60 * 60 * 1000, // one day
 		ignoreModules: [],
-		sendUpdatesNotifications: false
+		sendUpdatesNotifications: false,
+		updates: [
+			// array of module update commands
```

Comment:
The defaults should be empty, better put this into the sample config and the documentation

Reply:
sure it's will be cleaned ;)
It's only for you understand how it can works (for develop)
It will be cleaned to `updates: []` when released

Reply:
Done